qemu/qmp.go

```diff
@@ -178,6 +178,28 @@ type MemoryDevices struct {
 	Type string            `json:"type"`
 }
 
+// CPUInfo represents information about each virtual CPU
+type CPUInfo struct {
+	CPU      int           `json:"CPU"`
```

Comment:
No reason to have the json format being upper case. I would prefer ``json:"cpu"``

Reply:
I don't think we can modify the json format, we don't have the choice for json format, it's a QMP response which is defined by qemu.

Reference link:  https://qemu.weilnetz.de/doc/qemu-qmp-ref.html

```
Command: query-cpus
Returns a list of information about each virtual CPU.

This command causes vCPU threads to exit to userspace, which causes a small interruption to guest CPU execution. This will have a negative impact on realtime guests and other latency sensitive guest workloads. It is recommended to use query-cpus-fast instead of this command to avoid the vCPU interruption.

Returns: a list of CpuInfo for each virtual CPU

Since: 0.14.0

Example:

-> { "execute": "query-cpus" }
<- { "return": [
         {
            "CPU":0,
            "current":true,
            "halted":false,
            "qom_path":"/machine/unattached/device[0]",
            "arch":"x86",
            "pc":3227107138,
            "thread_id":3134
         },
         {
            "CPU":1,
            "current":false,
            "halted":true,
            "qom_path":"/machine/unattached/device[2]",
            "arch":"x86",
            "pc":7108165,
            "thread_id":3135
         }
      ]
   }
Notes: This interface is deprecated (since 2.12.0), and it is strongly recommended that you avoid using it. Use query-cpus-fast to obtain information about virtual CPUs.
```